### DISCUSSION TITLE
Why response body is not showing for failures

### DISCUSSION THREAD

**Discussion by mani9223-oss** [TwiN/gatus#445] [Q&A] [answered]
@TwiN Is it possible to show the response body for failures? Its not showing right now

**Comment by mani9223-oss** [comment]
@TwiN Is it possible to show response body for failures?

**Comment by TwiN** [comment, marked as answer]
You can use the `[BODY]` placeholder to check for a specific value in the body, and if the specific value is not found, the actual value will be shown (depending on the length of the body, it may be truncated)

Here's an example: 
![image](https://user-images.githubusercontent.com/15699766/223619009-67ff18d7-9883-4879-971b-aac04f1cfe95.png)

**Reply by FahDon** [reply]
@TwiN I think there are some scenarios where the Body doesn't match the defined JSON structure. Below is an example.
 
![Screenshot 2023-03-19 at 11 24 50 pm](https://user-images.githubusercontent.com/1430566/226207023-2aad8453-53e4-4f45-915a-36bfd75e0a73.jpg)

OP raised a good point, it would be useful to check the full body in case of failure.
